### PR TITLE
build(): generate a <script> version of the sdk

### PR DESCRIPTION
**Pull request by taymoork2**
# Pull Request Template

## Description

This is to simplify the process of using the client in the browser environment
Created a rollupjs config that bundles a newly created browser.js file into a ~664kb script that can be used in a non SPA <html> environment.

```javascript
// browser.js

var CiscoSpark = window.ciscospark = require('ciscospark');

module.exports = CiscoSpark;
```

TODO

- [ ] Setup tests to verify that it doesn't break
- [ ] Setup Jenkins job to build
- [ ] Update docs on developer portal

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,8 +44,10 @@ You will need to set the following environment variable:
 
 ```javascript
 const assert = require(`assert`);
-assert(process.env.CISCOSPARK_ACCESS_TOKEN, 'This example assumes you have set your access token as an environment variable');
 const ciscospark = require(`ciscospark`);
+
```
Comment (mattnorris) [inline README.md]: Any reason to move this statement?

File: README.md [hunk 2]
```diff
@@ -102,13 +102,13 @@ You'll be able to load the samples by visiting `https://localhost:8000/packages/
 
 ### Available Samples
 
-| Sample | App Link | Source |
-| ------ | -------- | ------ |
-| Implicit Grant Flow | [local app](https://localhost:8000/packages/node_modules/samples/browser-auth-implicit) | [code](./packages/node_modules/samples/browser-auth-implicit) |
-| Single Party Calling | [local app](https://localhost:8000/packages/node_modules/samples/browser-single-party-call) | [code](./packages/node_modules/samples/browser-single-party-call) |
+| Sample                         | App Link                                                                                              | Source                                                                      |
```
Comment (mattnorris) [inline README.md]: Any reason for these changes?

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,11003 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (mattnorris) [inline yarn.lock]: I'd prefer not to introduce more tooling. The newest versions of npm can achieve the same things as yarn.

File: README.md (hunk 1 above)
Comment (taymoork2) [reply README.md]: Moved it by accident

File: README.md (hunk 2 above)
Comment (taymoork2) [reply README.md]: I think my linter went ahead and autofixed when i saved

File: package.json [hunk 4]
```diff
@@ -201,6 +202,15 @@
     "require-dir": "^0.3.1",
     "response-time": "^2.3.2",
     "rimraf": "^2.6.1",
+    "rollup": "^0.59.3",
```
[the remark this answers is not in the document]
Comment (taymoork2) [reply package.json]: is how I installed them in the beginning, but there isn't a devDep section in the package.json
More than happy to move them to devDeps, might need to move more over into that section too

File: tooling/commands/build.js [hunk 5]
```diff
@@ -30,6 +30,9 @@ module.exports = {
         for (const pName of await list()) {
           await buildPackage(pName);
         }
+
+        // only build script after packages are built and only when all packages are buing built
```
Comment (mattnorris) [inline tooling/commands/build.js]: - "being built"
- Is the script built each time a package is built?

**Pull request merged**